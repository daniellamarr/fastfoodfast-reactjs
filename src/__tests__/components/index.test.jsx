import 'babel-polyfill';
import React from 'react';
import { shallow } from 'enzyme';
import Container from '../../routes/index.jsx';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Container />);
});

describe('Container Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
