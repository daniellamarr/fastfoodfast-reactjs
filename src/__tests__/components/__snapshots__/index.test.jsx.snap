// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Component should match snapshot 1`] = `
<div>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/signup"
    />
    <Route
      component={[Function]}
      path="/login"
    />
    <Route
      component={[Function]}
      path="/admin/login"
    />
    <Route
      component={[Function]}
      path="/profile"
    />
    <Route
      component={[Function]}
      path="/menu/:id"
    />
    <Route
      component={[Function]}
      path="/checkout"
    />
  </Switch>
</div>
`;
